pipeline/schema/schema.go

```diff
@@ -0,0 +1,32 @@
+package schema
+
+import (
+	_ "embed"
+	"fmt"
+
+	"github.com/woodpecker-ci/woodpecker/shared/yml"
+	"github.com/xeipuuv/gojsonschema"
+)
+
+//go:embed schema.json
```

Comment:
Go  min version 1.16 - we should put a node somewhere

Reply:
Maybe in the development guide (#314) with a list of required tools to develop woodpecker.

Reply:
updated mentioned issue